src/openrct2/interface/window.h

```diff
@@ -497,6 +497,7 @@ enum {
 #define WC_TOP_TOOLBAR__WIDX_WATER                      9
 #define WC_TOP_TOOLBAR__WIDX_SCENERY                    10
 #define WC_TOP_TOOLBAR__WIDX_PATH                       11
+#define WC_TOP_TOOLBAR__WIDX_CLEAR_SCENERY              17
```

Comment:
`validate_global_widx(WC_TOP_TOOLBAR, WIDX_CLEAR_SCENERY)` has been added to `top_toolbar.c`